<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index</class>
 <widget class="QWidget" name="Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="navigate_sa">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="widgetLeft">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>178</width>
        <height>542</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <!-- <widget class="QWidget" name="myFiles_page"/> -->
     <!-- <widget class="QWidget" name="shareFile_page"/> -->
     <!-- <widget class="QWidget" name="transferFile_page"/> -->
     <!-- <widget class="QWidget" name="trashFile_page"/> -->
     <!-- <widget class="QWidget" name="myInfo_page"/> -->
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
